### PR TITLE
Update filtering docs

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: docs/files/tutorial-get.py [hunk 1]
```diff
@@ -20,8 +20,8 @@
 # Filter by using several fields:
 res.data.filter(
   status='OK',
-  created__gt='2018-10-01',
-  created__lt='2025-11-01',
+  created__gt='2018-10-01 01:23:45',
```
Comment (dblenkus) [inline docs/files/tutorial-get.py:23]: Why don't you use `datetime.datetime(2018, 10, 1)`? It looks more natural and easier to remember.

Comment (JureZmrzlikar) [reply docs/files/tutorial-get.py]: Done.

File: docs/tutorial-get.rst [hunk 2]
```diff
@@ -63,7 +63,7 @@ But the real power of the ``filter()`` method is in combining multiple
 parameters:
 
 .. literalinclude:: files/tutorial-get.py
-  :lines: 20-27
+  :lines: 20-29
```
Comment (dblenkus) [inline docs/tutorial-get.rst:66]: Does this also break all other references further down the file?

Comment (JureZmrzlikar) [reply docs/tutorial-get.rst:66]: Yes :-)

**Pull request merged**